src/modules/Security.rb

```diff
@@ -314,13 +314,7 @@ def inittab_shutdown_configured?
     # Read the information about ctrl+alt+del behavior
     # See bug 742783 for description
     def ReadConsoleShutdown
-      ret = ::Security::CtrlAltDelConfig.current || ::Security::CtrlAltDelConfig.default
-
-      return ret if ::Security::CtrlAltDelConfig.systemd?
-
-      @Settings["CONSOLE_SHUTDOWN"] = ret if ::Security::CtrlAltDelConfig.inittab?
-
-      nil
+      ::Security::CtrlAltDelConfig.current || ::Security::CtrlAltDelConfig.default
```

Comment:
Why you discarded this solution?.

It could be something like:

```ruby
def ReadConsoleShutdown
  @Settings["CONSOLE_SHUTDOWN"] = ::Security::CtrlAltDelConfig.current || ::Security::CtrlAltDelConfig.default

  nil
end
```

And in Read just modify:
```ruby
@Settings["CONSOLE_SHUTDOWN"] = ReadConsoleShudtown()
```
by
```ruby
ReadConsoleShudtown()
```

... just to know the reasons behind the change.